tpl/bootstrap-compact.php

```diff
@@ -5,7 +5,7 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<meta name="robots" content="noindex" />
-		<title><?php echo i18n::_('PrivateBin'); ?></title>
+		<title><?php echo PrivateBin\i18n::_('PrivateBin'); ?></title>
```

Comment:
Question: Could we add a `use PrivateBin` on top of the template or in the `view` class and then use `i18n` without namespace?